src/index.ts

```diff
@@ -89,7 +112,38 @@ async function buildDenoLambda(
   try {
     await execa(
       path.join(workPath, 'layer', 'bin', 'deno'),
-      ['bundle', entrypointPath, binPath, ...(unstable ? ['--unstable'] : [])],
+      [
+        'bundle',
+        entrypointPath,
+        binPath,
+        // Boolean
+        ...(unstable ? ['--unstable'] : []),
+        ...(allowAll ? ['-A'] : []),
+        ...(allowEnv ? ['--allow-env'] : []),
+        ...(allowRun ? ['--allow-run'] : []),
+        ...(allowHrTime ? ['--allow-hrtime'] : []),
+
+        // Boolean or Strings (only one of each pair will be applied)
+        ...(allowRead && typeof allowRead === 'boolean'
+          ? ['--allow-read']
+          : []),
+        ...(allowRead && typeof allowRead === 'string'
+          ? [`--allow-read=${allowRead}`]
+          : []),
+        ...(allowWrite && typeof allowWrite === 'boolean'
+          ? ['--allow-write']
+          : []),
+        ...(allowWrite && typeof allowWrite === 'string'
+          ? [`--allow-write=${allowWrite}`]
+          : []),
+        ...(allowNet && typeof allowNet === 'boolean' ? ['--allow-net'] : []),
+        ...(allowNet && typeof allowNet === 'string'
+          ? [`--allow-net=${allowNet}`]
+          : []),
+
+        // Strings
+        ...(tsConfig ? ['-c', tsConfig] : []),
```

Comment:
--config flag is only available from 1.0.0 upwards.
Consider adding a version check to prevent unknown flag error.

Reply:
good catch, ty

Reply:
Fixed